### DISCUSSION TITLE
Welcome to codon Discussions!

### DISCUSSION THREAD

**Discussion by arshajii** [exaloop/codon#65] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by stucotso** [comment]
Hi there! Really impressed by your project here.  I'd love to contribute - first thing seems to be potentially a Windows port. Is this in progress already? Any ideas yet on how much work to get Windows binaries?   Have you also considered mobile yet and what might be needed there?

Great work here, this might be the first high-performance Python implementation I would use as my main go-to!  Once platform support is there.

**Comment by setop** [comment]
This is the most accomplished AOT/JIT Python compiler I've ever worked with.
Enjoy the expressiveness of Python, add some typing hints if asked for, compile and you get 10x performance boost, almost the efficiency of native code.
I've just scratch the surface. I want to play with C integration, parallelism, concurrency, GPU acceleration, and all other features.

Thank for this amazing work.

**Comment by Priyanshu1045** [comment]
Actually i'm a beginner in c++ but i want to contribute in this project

**Comment by iuridiniz** [comment]
# This language can easily be a game changer

I'm really impressed with this project. It is an amazing idea.

Here we have 3 amazing ingredients:
- The ease of python syntax.
- The speed of C++.
- The safety provided by a garbage collector. 

With the emergence of safer and faster languages ​​(Rust, Golang, Julia, among others), why not look for something that is easy and safe like python, but faster? Congratulations.

I hope this language can be on par with Golang or maybe better than it.

----
**About me**: Senior developer with a passion for the embedded world who lives in Brazil.

**Comment by Priyanshu1045** [comment]
Thank you so much sir for your advice
This can help me for growing my future as i'm a very beginner I need more
guidence like this hope to get that from you and your team
Regards
